api/src/main/resources/messages.properties

```diff
@@ -2165,6 +2165,7 @@ User.useExisting=Use a person who already exists
 User.whichPerson=Which person?
 User.delete.failure=Cannot delete user. There is data attributed to this user
 User.delete.success=Successfully deleted user.
+User.DemographicInfo.length=Error in Demographic Info fields. The maximum allowed length of each field is {0}.
```

Comment:
thanks @Parth59  it would be nice to have all this in one single pr that uses the error message